tests/test_mcmc.py

```diff
@@ -381,6 +381,29 @@ def test_global_signal(lc_core, lc_core_ctx):
     assert "frequencies" in model
 
 
+def test_edges_timing_only(lc_core, lc_core_ctx):
```

Comment:
this test will need to be modified if you use only the single class with an option, as I mentioned above.